Index: src/main/java/org/Game/UI.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.Game;\r\n\r\nimport java.awt.*;\r\nimport java.text.DecimalFormat;\r\n\r\npublic class UI {\r\n    GamePanel gp;\r\n    Font arial_40;\r\n\r\n    double playtime;\r\n    DecimalFormat dFormat = new DecimalFormat(\"#0.00\");\r\n\r\n    public UI(GamePanel gp)\r\n    {\r\n        this.gp = gp;\r\n        arial_40 = new Font(\"Arial\", Font.PLAIN, 40);\r\n    }\r\n\r\n    public void draw(Graphics2D g2)\r\n    {\r\n        g2.setFont(arial_40);\r\n        g2.setColor(Color.white);\r\n\r\n        // TIME\r\n        playtime += (double)1/gp.FPS;\r\n        g2.drawString(\"Time: \" + dFormat.format(playtime), gp.tileSize*11, 65);\r\n\r\n\r\n\r\n\r\n    }\r\n}\r\n
===================================================================
diff --git a/src/main/java/org/Game/UI.java b/src/main/java/org/Game/UI.java
--- a/src/main/java/org/Game/UI.java	(revision 385aca45bc9fdd25957b4188e43847d19c671975)
+++ b/src/main/java/org/Game/UI.java	(date 1745370846210)
@@ -25,8 +25,5 @@
         playtime += (double)1/gp.FPS;
         g2.drawString("Time: " + dFormat.format(playtime), gp.tileSize*11, 65);
 
-
-
-
     }
 }
